app/dashboards/member_dashboard.rb

```diff
@@ -0,0 +1,62 @@
+require "administrate/base_dashboard"
+
+class MemberDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    events: Field::HasMany,
+    organizations: Field::HasMany,
+    user: Field::BelongsTo,
+    id: Field::Number,
+    name: Field::String,
+    email: Field::String,
+    picture: Field::String.with_options(searchable: false),
+    twitter_handle: Field::String,
+    github_handle: Field::String,
+    biography: Field::Text,
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime,
+  }
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = [
+    :events,
+    :organizations,
+    :user,
```

Comment:
@Coaxial, please: 
- Rebase with master and fix the conflicts.
- Don't replace `:user` with `:name` that was a hack to get it to work, do the right thing ;)
- Get familar with the user model :)
- It's better if you remove the `user_dashboard` and generate a new one based on the last changes like so: `rails g administrate:dashboard user`
- Let me know if you need a hand to finish this